CHANGELOG

```diff
@@ -1,3 +1,7 @@
+0.8.0
+=====
+- Add flag to allow aws detection to pass when no keys exist
+
```

Comment:
Don't bother with the changelog, I'll fill this out when I decide to make a release <3